src/redesign/login/LoginFailure.jsx

```diff
@@ -73,7 +73,7 @@ const LoginFailureMessage = (props) => {
       break;
     case FAILED_LOGIN_ATTEMPT: {
       resetLink = (
-        <a href="/reset">
+        <a href="/reset" style={{ 'text-decoration': 'underline' }}>
```

Comment:
nit: We can use [Hyperlink](use https://paragon-edx.netlify.app/components/hyperlink/) here (and wherever required) with its prop `isInline` set to true